Update README (#2549)
gitpod-io/gitpod: README.md

## Features

>  - Dev environments as code

Author: **thought**: Maybe it's worth taking a fresh look at this list and update accordingly. Listing distinct features could also help.

> Open any GitLab, GitHub, or Bitbucket project, by prefixing the project's URL with `gitpod.io/#`. 
> 
> E.g. https://gitpod.io/#https://github.com/gitpod-io/gitpod

Author: **thought**: We could add a more expanded getting started section, following the update in the [getting started section in docs](https://www.gitpod.io/docs/getting-started/).

```suggestion
You can start using Gitpod with one or more of the following ways:

1. [Use a Prefixed URL](https://www.gitpod.io/docs/context-urls/)
1. [Install Browser Extension](https://www.gitpod.io/docs/browser-extension/)
1. [Enable GitLab Integration](https://www.gitpod.io/docs/gitlab-integration/)
1. Quick start using an [Example Project](https://www.gitpod.io/docs/getting-started/#example-project) or [OSS Project](https://www.gitpod.io/docs/getting-started/#gitpodified-open-source-project)
```

## Documentation

All documentation can be found on https://docs.gitpod.io.
For example, see [Introduction](https://www.gitpod.io/docs/) and [Getting Started](https://www.gitpod.io/docs/getting-started/) sections. 📚

## Questions

> For questions and support please use the [community forum](http://community.gitpod.io/).

Reviewer: Maybe we could specifically mention gitpod.io here, too. My hope would be that we can bring down the number of issues due to service issues.

Author: This was also the intent behind [this proposal](https://typefox.slack.com/archives/C01APSL1RPV/p1605108490052500) (internal).

Join the conversation, and connect with other community members. 💬

You can also follow [`@gitpod`](https://twitter.com/gitpod) for announcements and updates from our team.

## Issues

> The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️

Author: **nitpick**: Probably worth adding a link to the issue tracker.

```suggestion
The [issue tracker](https://github.com/gitpod-io/gitpod/issues) is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️
```

> The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️

Reviewer 2: ```suggestion
The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future development efforts.
``` 
:stuck_out_tongue:

Author: Thanks for taking a look @ArthurSens! Do you also suggest removing the 🗺️ emoji from there?

Reviewer 2: Whoopis, no! that was an accident :sweat_smile:

You can upvote [popular feature requests](https://github.com/gitpod-io/gitpod/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) or [create a new one](https://github.com/gitpod-io/gitpod/issues/new?template=feature_request.md).

> ## Related Projects

Author: **question**: Maybe add a reference for these projects once we move these to `gitpod-io` org? /cc @csweichel

During the development of Gitpod we had to also develop our own tooling infrastructure to make development easier and more efficient.

> To this extend we've developed a number of open source projects including [Werft](https://github.com/csweichel/werft), [Leeway](https://github.com/TypeFox/leeway), and [Dazzle](https://github.com/csweichel/dazzle/).

Author: Added in https://github.com/gitpod-io/gitpod/pull/2549/commits/2ddf3d17ed758586ebc8b52fdc207eb421c7e6ef using the updated links.
